alacritty/src/config/window.rs

```diff
@@ -137,6 +137,16 @@ impl WindowConfig {
     pub fn maximized(&self) -> bool {
         self.startup_mode == StartupMode::Maximized
     }
+
+    #[cfg(target_os = "macos")]
+    pub fn option_as_alt(&self) -> winit::platform::macos::OptionAsAlt {
+        match self.option_as_alt {
+            OptionAsAlt::OnlyLeft => winit::platform::macos::OptionAsAlt::OnlyLeft,
+            OptionAsAlt::OnlyRight => winit::platform::macos::OptionAsAlt::OnlyRight,
+            OptionAsAlt::Both => winit::platform::macos::OptionAsAlt::Both,
+            OptionAsAlt::None => winit::platform::macos::OptionAsAlt::None,
```

Comment:
Shouldn't use the full path here. Just import it as `WinitOptionsAsAlt` or something.